src/pages/gear.page.tsx

```diff
@@ -1,11 +1,85 @@
+import { useEffect, useState } from 'react'
+import { ParchmentInput } from '../components/ParchmentInput'
 import { PageHeader } from '../components/page-header'
 import { Parchment } from '../components/parchment'
 import { rollD6 } from '../functions/dice.functions'
+import { ParchmentButton } from '../components/ParchmentButton'
+import { ArrowPathIcon } from '@heroicons/react/24/outline'
+import { t } from 'i18next'
+import { useTranslation } from 'react-i18next'
 
 export const GearPage = () => {
+  const { t, i18n } = useTranslation(['gear'])
+  const [searchGoods, setSearchGoods] = useState('')
+  const [maxPrice, setMaxPrice] = useState<number>(Infinity)
+  const [itemAvailability, setItemAvailability] = useState<Record<
+    string,
+    Availability
+  > | null>(null)
```

Comment:
I like to use `undefined` instead of `null`.

Maybe this should be using `useLocalStorage` to save the Availability between session?